ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/MLModelTool.java

```diff
@@ -8,6 +8,8 @@
 import java.util.List;
 import java.util.Map;
 
+import javax.swing.*;
+
```

Comment:
Why is this import required?

Reply:
And why is it a `*`?  Wildcard imports are generally discouraged.